cogboard-webapp/src/components/Widget/index.js

```diff
@@ -109,7 +110,7 @@ const Widget = ({ id, index }) => {
   return (
     <>
       <StyledCard
-        status={status}
```

Comment:
have you removed this constant ? `status`

Reply:
Good catch, fix will be in the next commit.